src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
The [example you provided](https://github.com/graphcool/graphql-config/pull/4/files/6d36a2ceb8c6dc1ea6ab1405179b2728dbf6020e#r74434027) gave a file path with a leading slash and you said it should resolve relative to the cwd (`/relay/data/schema.json`). I am saying that I think a file path with a leading slash should be absolute to the filesystem and a file without a leading slash should be relative. That is the way most APIs (`require`, for instance) work.